### DISCUSSION TITLE
Use multiple monitors

### DISCUSSION THREAD

**Discussion by mshzsh** [hyprwm/Hyprland#750] [Q&A]
I have `hyperland` installed on my laptop and when I connect an external monitor it does not work, and even inside `arandr` does not recognize it while  `wlr-randr` show monitor name, the laptop graphic card is Nvidia and I did all the settings and when I switch to `bspwm` everything works properly

**Comment by mshzsh** [comment]
wlr-randr output :

```
eDP-1 "LG Display 0x04B9 (eDP-1)"
  Physical size: 340x190 mm
  Enabled: yes
  Modes:
    1920x1080 px, 60.020000 Hz (preferred, current)
  Position: 0,0
  Transform: normal
  Scale: 1.000000
HDMI-A-4 "Samsung Electric Company SAMSUNG 0x00000700 (HDMI-A-4)"
  Physical size: 1110x620 mm
  Enabled: no
  Modes:
    640x480 px, 59.939999 Hz
    .
    .
    .
    3840x2160 px, 30.000000 Hz (preferred)
```

hyprland.conf : 

```
monitor=eDP-1,1920x1080@60,0x0,1
monitor=HDMI-A-4,1920x1080@60,1920x0,1

```

**Reply by taylor85345** [reply]
This is a known issue with Nvidia Optimus hybrid graphics laptops that affects all wlroots-based compositors. There are some potential workarounds [in the wiki](https://wiki.hyprland.org/FAQ/#my-external-monitor-is-blank--doesnt-render--receives-no-signal-laptop), but no great solutions at this time.

As a side note, arandr is an X11 utility and won't work properly under Wayland. wdisplays is a similar graphical utility for wlroots, but it is typically advised to do as much configuration as possible in the hyprland config.

**Reply by mshzsh** [reply]
Hopefully one day Nvidia will be properly supported in wayland.
Thank you!
